model-optimizer/extensions/front/interpolate_reshape.py

```diff
@@ -119,10 +119,13 @@ def get_non_interpolate_concat_sources(self, concat: Node):
         or through identity operation sequence. Returns the list of Concat sources that satisfy the condition.
         """
         assert concat.soft_get('type') == 'Concat'
-        sources = []
+        sources, ports_to_omit = [], []
+        if concat.has_valid('N'):
+            # TODO: should be removed after Concat operation normalization
+            ports_to_omit.append(concat.N)
```

Comment:
Please, create a task in the backlog 12341 to normalize the Concat operation and get rid of `op='ConcatV2'` from the MO completely.